graal/graal_bindist.bzl

```diff
@@ -41,6 +41,10 @@ def _graal_bindist_repository_impl(ctx):
         sha256 = sha,
         stripPrefix = archive_internal_prefix,
     )
+    exec_result = ctx.execute(["bin/gu", "install", "native-image"], quiet=False)
```

Comment:
I was so reluctant to do it this way. This requires network access I think. I think there is a flag to gu that you can use to pass the download of native image.

Reply:
Other than if you hit the repo cache for http. your doing network here already right? its just the repo rule

Reply:
but i'm peaceful with doing a manual download of the native image stuff and then telling it about it, so it could all be cached?

Reply:
Is it hash protected without? My fear would be gu just fetches some latest version.

Reply:
thats a fair point, i'll have to look into that

Reply:
I agree with Oscar, but considering that this ruleset is in the early stages, I won't block this PR over it.